src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -24,6 +24,7 @@
 using Microsoft.Health.Fhir.Core.Features.Search;
 using Microsoft.Health.Fhir.Core.Features.Validation;
 using Microsoft.Net.Http.Headers;
+using UnsupportedMediaTypeException = Microsoft.Health.Abstractions.Exceptions.UnsupportedMediaTypeException;
```

Comment:
Is there a conflict?

Reply:
Oops. There was when I first pulled the branch, but not any longer. I'll update the using.